gnovm/stdlibs/hash/marshal_test.gno

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Test that the hashes in the standard library implement
+// BinaryMarshaler, BinaryUnmarshaler,
+// and lock in the current representations.
+
+package hash
+
+import (
+	"bytes"
+	"crypto/md5"
```

Comment:
Not directly related to this particular pull request, but if I remember correctly, we had discussed completely prohibiting this import at some stage.

cc @jaekwon

Reply:
It's okay actualy.